wp-graphql.php

```diff
@@ -32,27 +32,59 @@
 	require_once __DIR__ . '/c3.php';
 }
 
-// Whether to autoload the files or not.
-// This must be defined here and not within the WPGraphQL.php because this constant
-// determines whether to autoload classes or not
-if ( ! defined( 'WPGRAPHQL_AUTOLOAD' ) ) {
-	define( 'WPGRAPHQL_AUTOLOAD', true );
-}
+/**
+ * Load files that are required even if the composer autoloader isn't installed
+ *
+ * @return void
+ */
+function graphql_require_bootstrap_files(): void {
 
-// Run this function when WPGraphQL is de-activated
-register_deactivation_hook( __FILE__, 'graphql_deactivation_callback' );
-register_activation_hook( __FILE__, 'graphql_activation_callback' );
+	$composer_file   = __DIR__ . '/composer.json';
+	$composer_config = null;
 
-// Bootstrap the plugin
-if ( ! class_exists( 'WPGraphQL' ) ) {
-	require_once __DIR__ . '/src/WPGraphQL.php';
+	if ( file_exists( $composer_file ) ) {
+		// @phpcs:ignore
+		$contents        = file_get_contents( $composer_file );
+		$composer_config = ! empty( $contents ) ? json_decode( $contents ) : null;
+	}
+
+	if ( isset( $composer_config->autoload->files ) ) {
+		foreach ( $composer_config->autoload->files as $file ) {
+			$file_path = __DIR__ . '/' . $file;
+			if ( file_exists( $file_path ) ) {
+				// @phpcs:ignore
+				require_once $file_path;
+			}
+		}
+	}
```

Comment:
It's only a few files, can we `require_once` them manually instead of pulling in + parsing a json file?

( would also solve `composer.json` including `src/WPGraphQL` both in `files` and in `psr-4` )

Reply:
ya, makes sense. I did have it this way in one commit, then reversed it. Seems sensible to re-reverse back to the manual require_once approach.